internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -319,7 +319,7 @@ public RssSendShuffleDataResponse sendShuffleData(RssSendShuffleDataRequest requ
               .setLength(sbi.getLength())
               .setTaskAttemptId(sbi.getTaskAttemptId())
               .setUncompressLength(sbi.getUncompressLength())
-              .setData(ByteString.copyFrom(sbi.getData()))
+              .setData(ByteString.copyFrom(sbi.getData().nioBuffer()))
```

Comment:
Could we reduce data copy here?

Reply:
ByteBuf.nioBuffer() will not cause a copy of the byte array, So there is no difference from before the modification.

Reply:
setData needs to pass in a parameter value of type ByteString, which is a type of protobuf, which doesn’t seem to work.

Reply:
Looks like we can use NioByteString, I'll test it out.

Reply:
Okay, I will replace it with UnsafeByteOperations.unsafeWrap().